OpenRA.Mods.Common/UpdateRules/Rules/20200202/RemoveRestrictMCVDeploymentFallbackToBase.cs

```diff
@@ -0,0 +1,38 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class RemoveRestrictMCVDeploymentFallbackToBase : UpdateRule
+	{
+		public override string Name { get { return "RestrictMCVDeploymentFallbackToBase trait has been removed from McvManagerBotModule."; } }
```

Comment:
Since `RestrictMCVDeploymentFallbackToBase` is a property of `McvManagerBotModule`, not a trait:
```suggestion
		public override string Name { get { return "RestrictMCVDeploymentFallbackToBase has been removed from McvManagerBotModule."; } }
```